src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/NodeBuilder.cs

```diff
@@ -183,5 +184,59 @@ public void Dispose()
             LogManager.Configuration.LoggingRules.Clear();
             LogManager.ReconfigExistingLoggers();
         }
+
+        /// <summary>
+        /// By default, logs are disabled when using <see cref="Create(string)"/> or <see cref="Create(object, string)"/> methods,
+        /// so using this fluent method, the caller can enable the logs at will.
+        /// </summary>
+        /// <returns>Current <see cref="NodeBuilder"/> instance, used for fluent API style</returns>
+        /// <example>
+        /// //default use (without logs)
+        /// using (NodeBuilder builder = NodeBuilder.Create(this))
+        /// {
+        ///     //your test code here
+        /// }
+        ///
+        /// //with logs enabled
+        /// using (NodeBuilder builder = NodeBuilder.Create(this).WithLogsEnabled())
+        /// {
+        ///     //your test code here
+        /// }
+        /// </example>
+        public NodeBuilder WithLogsEnabled()
+        {
+            if (!LogManager.IsLoggingEnabled())
```

Comment:
Can't we use 
```
            // NLog Enable/Disable logging is based on internal counter, so to be sure to have enabled logs, must to
            // keep calling EnableLogging until IsLoggingEnabled returns true.
            while (!LogManager.IsLoggingEnabled())
            {
                LogManager.EnableLogging();
            }
```

Instead of 
```
            if (!LogManager.IsLoggingEnabled())
            {
                // NLog Enable/Disable logging is based on internal counter, so to be sure to have enabled logs, must to
                // keep calling EnableLogging until IsLoggingEnabled returns true.
                do
                {
                    LogManager.EnableLogging();
                } while (!LogManager.IsLoggingEnabled());
            }
```

Reply:
Same with `WithLogsDisabled`

Reply:
yes I already though of that when i was already in the bed lol, will change tonight with other corrections